### ISSUE TITLE
Convert shows tab to be native

### ISSUE THREAD

**Issue by orta** [artsy/eigen#853]
We can ship the raw Json with the app - it's not changed in a long time and we can save on a top level wait on martsy.

**Comment by alloy** [comment]
Can you be more specific what screen this is about? Is it the list of cities?

**Comment by orta** [comment]
I can go one better, I have a little bit of time.

**Comment by orta** [comment]
https://github.com/artsy/eigen/compare/cc23d390494403fe4f413c72af4742e07fbda36d...orta-native_show_locations

http://cl.ly/3L3q1A130h2U

**Comment by alloy** [comment]
Nice!

Am I correct in understanding that there’s no API for this JSON atm or is there?

**Comment by orta** [comment]
it's located at https://github.com/artsy/places - Ideally a more wrapped up version would include a link to that in the source code. As well as a random show image like they do in the header for a martsy view:

<img width="616" alt="screen shot 2015-10-06 at 13 14 42" src="https://cloud.githubusercontent.com/assets/49038/10307105/38465980-6c2c-11e5-8dec-0cf7f9c90ae2.png">

**Comment by alloy** [comment]
Aye, agreed.

**Comment by alloy** [comment]
I would like to see us get rid of all webviews that show countries/cities and replace them with native ones. Additionally we can add a feature to use the user’s physical location to show the nearest one.

**Comment by katarinabatina** [comment]
Not a priority for this sprint, or in general since we'll be bumping this out of eigen
